openmc/__init__.py

```diff
@@ -1,3 +1,5 @@
+from openmc.cell import *
```

Comment:
Oh nvm, I see that you've broken out `Cell` and `Lattice` into their own source files. This was a much needed change, I'm glad to finally see it in place!